global.ini

```diff
@@ -8128,279 +8128,279 @@ Heads_Male_06=Gesicht 06
 Heads_Male_07=Gesicht 07
 Heads_Male_08=Gesicht 08
 Heads_Male_09=Gesicht 09
-Hints_ADS1=Verwenden Sie ~action(player|zoom), um auf das Visier der Waffe zu zielen.
-Hints_ADSContext01=Um den Atem anzuhalten, verwenden Sie ~action(player|stabilize).\nZum Zoom In/Out verwenden Sie ~action(player|zoom_in) / ~action(player|zoom_out).
-Hints_ADSContext01_Gamepad=Um den Atem anzuhalten, verwenden Sie ~action(player|stabilize).\nZum Zoomen verwenden Sie ~action(player|zoom_in_out).
-Hints_ASOP1=Fahrzeugauswahl-Kioske ermöglichen es Ihnen, jedes Ihrer Schiffe mit dem Flottenmanager abzurufen.\nSie finden sie in der Mitte von Deck 1.
-Hints_ASOP2=Passieren Sie durch eine Luftschleuse, um die Landeplätze zu erreichen.\nFolgen Sie dem AR-Wegpunkt auf Ihrem Visier, um Ihr Schiff zu lokalisieren.
-Hints_ASOP3=Aktivieren Sie den Fahrzeugauswahlkiosk mit Interaktionsmodus (~action(player_choice|pc_interaction_mode)).\nNavigieren Sie im Flottenmanager und wählen Sie „Abrufen“ auf dem Schiff, das Sie verwenden möchten.
+Hints_ADS1=Verwende ~action(player|zoom), um auf das Visier der Waffe zu zielen.
+Hints_ADSContext01=Um den Atem anzuhalten, verwende ~action(player|stabilize).\nZum Zoomen In/Out verwende ~action(player|zoom_in) / ~action(player|zoom_out).
```

Comment:
In out könnte dann doch weg oder?

Reply:
Das wäre die Frage was der Schlüssel ausgibt. Hm

Reply:
Habs umgeschrieben